traffic_ops/app/script/update_riak_for_search.pl

```diff
@@ -46,9 +46,6 @@
 		}
 		$record->{deliveryservice} = $xml_id;
 		$record->{cdn} = $cdn;
-		$record->{certificate}->{crt} = decode_base64($record->{certificate}->{crt});
```

Comment:
This is an example of a client that would break, if this particular API is being used by a client outside of our codebase.